agent/engine/serviceconnect/manager_other.go

```diff
@@ -69,3 +69,11 @@ func (*manager) GetLoadedImageName() (string, error) {
 		"appnetAgent container get image name: unsupported platform: %s/%s",
 		runtime.GOOS, runtime.GOARCH))
 }
+
+func (*manager) GetLoadedAppnetVersion() string {
+	return ""
```

Comment:
Q. should we also return a `NewUnsupportedPlatformError` here?

Reply:
sure, will make this change!